### PR TITLE
VS-1310 Optionally convert filtered genotypes to no-calls

### PR DESCRIPTION
**Pull request by gbggrant**
This PR modifies the behavior of GvsExtractToPgen to no-call any filtered genotypes
It also allows one to run GvsExtractCallset so that VCFs generated by it also have no-called GTs.
I also took the liberty of renaming 'VQSR Classic' to 'VQSR' and 'VQSR Lite' to 'VETS' in much of the Java code.

Passing Integration test [here](https://app.terra.bio/#workspaces/gvs-dev/GVS%20Integration/job_history/1e1ed014-47cf-4c95-96f4-5c1284fc4616)
Run of tie out pgen to VCF with no-called GTs [here](https://app.terra.bio/#workspaces/gvs-dev/GVS%20Quickstart%20v3%20ggrant/job_history/9e78f44a-f531-450b-acd8-db66cc6454be).

### CODE REVIEW

File: scripts/variantstore/wdl/GvsExtractCallset.wdl [hunk 1]
```diff
@@ -44,8 +44,8 @@ workflow GvsExtractCallset {
     Int? split_intervals_mem_override
     Float x_bed_weight_scaling = 4
     Float y_bed_weight_scaling = 4
-    Boolean write_cost_to_db = true
     Boolean is_wgs = true
+    Boolean convert_filtered_genotypes_to_nocalls = false
```
Comment (gbggrant) [inline scripts/variantstore/wdl/GvsExtractCallset.wdl:48]: Although not required by the corresponding Jira, I made this functionality available via the wdl - largely to allow me to test, but it seems like a possibly useful feature.

File: scripts/variantstore/wdl/GvsTieoutPgenToVcf.wdl [hunk 2]
```diff
@@ -159,6 +159,8 @@ task Tieout {
     >>>
     output {
         File? tarball = "tieout.tgz"
+        Array[File] vcfs = glob("tieout/vcf/vcf_compare_chr*")
```
Comment (gbggrant) [inline scripts/variantstore/wdl/GvsTieoutPgenToVcf.wdl:162]: Making these arrays of VCFs available as outputs in order to allow me to manually verify that the gts are no-called.

File: src/main/java/org/broadinstitute/hellbender/tools/gvs/extract/ExtractCohort.java [hunk 3]
```diff
@@ -122,20 +122,28 @@ public enum VQScoreFilteringType {GENOTYPE, SITES, NONE}
...
+            doc = "If true, use VQSR scoring (vqs Lod score). Otherwise use VETS scoring (calibration_sensitivity)",
             optional = true
     )
-    private boolean isVQSRClassic = false;
+    private boolean isVQSR = false;
 
     @Argument(
-            fullName = "vqsr-score-filter-by-site",
-            doc = "If VQSR Score filtering is applied, it should be at a site level. Default is false",
+            fullName = "vqs-score-filter-by-site",
```
Comment (gbggrant) [inline src/main/java/org/broadinstitute/hellbender/tools/gvs/extract/ExtractCohort.java:132]: For parameters/things that are used for both VQSR and VETS, I'm using 'VQS'

File: scripts/variantstore/wdl/GvsExtractCallset.wdl [hunk 4]
```diff
@@ -235,6 +236,7 @@ workflow GvsExtractCallset {
         memory_gib                         = effective_extract_memory_gib,
         emit_pls                           = emit_pls,
         emit_ads                           = emit_ads,
+        convert_filtered_genotypes_to_nocalls = convert_filtered_genotypes_to_nocalls,
```
Comment (mcovarr) [inline scripts/variantstore/wdl/GvsExtractCallset.wdl:239]: Alas the reign of aligned whitespace has come to an end

File: src/main/java/org/broadinstitute/hellbender/tools/gvs/extract/ExtractCohort.java (hunk 3 above)
Comment (mcovarr) [reply src/main/java/org/broadinstitute/hellbender/tools/gvs/extract/ExtractCohort.java:132]: drop that in a comment for posterity?

**Review by mcovarr — APPROVED** [review]
Looks good! 👍

File: scripts/variantstore/wdl/GvsExtractCallset.wdl (hunk 1 above)
Comment (RoriCremer) [reply scripts/variantstore/wdl/GvsExtractCallset.wdl:48]: smart

File: src/main/java/org/broadinstitute/hellbender/tools/gvs/common/SchemaUtils.java [hunk 5]
```diff
@@ -76,7 +76,7 @@ public class SchemaUtils {
 
     public static final List<String> SAMPLE_FIELDS = Arrays.asList(SchemaUtils.SAMPLE_NAME_FIELD_NAME, SchemaUtils.SAMPLE_ID_FIELD_NAME);
     public static final List<String> YNG_FIELDS = Arrays.asList(FILTER_SET_NAME, LOCATION_FIELD_NAME, REF_ALLELE_FIELD_NAME, ALT_ALLELE_FIELD_NAME, VQSLOD, YNG_STATUS);
-    public static final List<String> VQSLITE_YNG_FIELDS = Arrays.asList(FILTER_SET_NAME, LOCATION_FIELD_NAME, REF_ALLELE_FIELD_NAME, ALT_ALLELE_FIELD_NAME, CALIBRATION_SENSITIVITY, SCORE, YNG_STATUS);
+    public static final List<String> VETS_YNG_FIELDS = Arrays.asList(FILTER_SET_NAME, LOCATION_FIELD_NAME, REF_ALLELE_FIELD_NAME, ALT_ALLELE_FIELD_NAME, CALIBRATION_SENSITIVITY, SCORE, YNG_STATUS);
```
Comment (RoriCremer) [inline src/main/java/org/broadinstitute/hellbender/tools/gvs/common/SchemaUtils.java:79]: thank youuuuuu

**Review by RoriCremer — APPROVED** [review]

**Pull request merged**